app/models/metadatum.rb

```diff
@@ -51,6 +57,21 @@ class Metadatum < ApplicationRecord
     sequencer: %w[MiSeq NextSeq HiSeq NovaSeq Other]
   }.freeze
 
+  # Mapping from alternative name to our name. Used at upload time.
+  KEY_ALT_NAMES = {
+    sample_unique_id: "unique_id",
+    sample_collection_date: "collection_date",
+    sample_collection_location: "collection_location",
+    :"rna/dna_input(ng)" => "rna_dna_input",
+    :"rna/dna_input (ng)" => "rna_dna_input",
+    :"rna/dna_input" => "rna_dna_input",
+    :"rna/dna input" => "rna_dna_input",
+    :"rna/dna input (ng)" => "rna_dna_input",
+    antibiotics_administered: "antibiotic_administered",
+    :"known_organism(s)" => "known_organism",
+    :known_organisms => "known_organism"
+  }.freeze
+
```

Comment:
Are we planning to (eventually) move all this configuration into db ?

Reply:
I think eventually yes but I kind of like it in code like this..